csharp-architectural-patterns/ModularMonolithPatternInCSharp/Common/RabbitMq/IRabbitMqConnectionManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RabbitMQ.Client;
+
+namespace Common.RabbitMq
+{
+    public interface IRabbitMqConnectionManager
+    {
+        IModel? Channel { get; set; }
+        IConnection? Connection { get; set; }
+
+        void Dispose();
```

Comment:
I suppose we don't need this declaration here.  Should come from IDisposable only?